src/sugar3/graphics/radiotoolbutton.py

```diff
@@ -16,9 +16,15 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
-"""
-STABLE.
-"""
+'''
+Provides a RadioToolButton class, similar to a "push" button.
+When the button is clicked, it depresses and is shaded darker.
+It is also possible to set a tooltip to be dispalyed when the user
+scrolls over it with their cursor.
+
+Example:
+    .. literalinclude ../examples/radiopalette.py
```

Comment:
That example is very old and doesn't even run

```
Traceback (most recent call last):
  File "radiopalette.py", line 13, in <module>
    box.pack_start(toolbar, False)
TypeError: Gtk.Box.pack_start() takes exactly 5 arguments (3 given)
```

Please make a more consise example of just the radiotoolbuttons, maybe looking at how they are used in the frame or homeview code